src/main/java/org/sagebionetworks/web/client/widget/entity/file/TableTitleBar.java

```diff
@@ -0,0 +1,84 @@
+package org.sagebionetworks.web.client.widget.entity.file;
+
+import org.sagebionetworks.repo.model.entitybundle.v2.EntityBundle;
+import org.sagebionetworks.repo.model.table.Dataset;
+import org.sagebionetworks.repo.model.table.Table;
+import org.sagebionetworks.web.client.widget.SynapseWidgetPresenter;
+import org.sagebionetworks.web.client.widget.entity.menu.v2.Action;
+import org.sagebionetworks.web.client.widget.entity.menu.v2.ActionMenuWidget;
+
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+
+public class TableTitleBar implements SynapseWidgetPresenter, TableTitleBarView.Presenter {
```

Comment:
Copied from FileTitleBar and removed everything that was File-specific, so this became very simple